xbmc/music/tags/MusicInfoTag.cpp

```diff
@@ -582,6 +582,7 @@ void CMusicInfoTag::SetArtist(const CArtist& artist)
   SetArtist(artist.strArtist);
   SetAlbumArtist(artist.strArtist);
   SetMusicBrainzArtistID({ artist.strMusicBrainzArtistID });
+  SetMusicBrainzAlbumArtistID({ artist.strMusicBrainzArtistID });
```

Comment:
To me it looks like SetMusicBrainzAlbumArtistID({ artist.strMusicBrainzArtistID }); and  SetAlbumArtist(artist.strArtist); should not be there or am I missing something?